src/routes/landing/components/FilterableCitySelector.js

```diff
@@ -45,9 +45,6 @@ class FilterableCitySelector extends React.Component<PropsType, StateType> {
   onFilterTextChange = (filterText: string) => this.setState({ filterText })
 
   _onPressLocationButton = () => {
-    if (Platform.OS === 'ios') {
-      Geolocation.requestAuthorization()
-    }
```

Comment:
The permissions are now requested in the IntroContainer if needed.

Reply:
Right, but if they are not given by the user, we still have a problem, right?

Reply:
I thought we want to disable the feature and show a snackbar in that case

Reply:
Afaics we should still display the GPS-Button for when something went wrong, or the user wants to use it anyways (even if he denied gps search in the slides).
If he presses the button I would expect to get a request for permission if I haven't done so yet. (Instead of nothing to happen when I press it)

Reply:
Okay, right, if there is a button you're definitely right. I've thought about it a little bit and I don't think we need the button at all. Possible cases:
- Feature activated and permissions granted: Definitely no button needed
- Feature activated and permissions not granted: The snack bar should be displayed (in a future task), which tells the user to grant permissions or disable the future, no button needed
- Feature deactivated: Definitely no button needed

So yeah, as long as there is no snackbar, I'll readd it, but I don't think we need the button in the future. Probably also makes https://issues.integreat-app.de/secure/RapidBoard.jspa?rapidView=14&view=detail&selectedIssue=NATIVE-393 redundant or at least affects it. @maxammann what do you think?

Reply:
For now I'll just readd it though.

Reply:
This is currently done by Judith :)